tests/ldevnet.go

```diff
@@ -21,7 +21,8 @@ func LaunchDevnetDocker(t *testing.T, numMiners int) *dockertest.Resource {
 		panic(fmt.Sprintf("couldn't create ipfs-pool: %s", err))
 	}
 	envNumMiners := fmt.Sprintf("TEXLOTUSDEVNET_NUMMINERS=%d", numMiners)
-	lotusDevnet, err := pool.RunWithOptions(&dockertest.RunOptions{Repository: "textile/lotus-devnet", Tag: "sha-9aab2c6", Env: []string{envNumMiners}, Mounts: []string{"/tmp/powergate:/tmp/powergate"}})
+	speed := "TEXLOTUSDEVNET_SPEED=500"
```

Comment:
I made devnets run a bit slower than default. This allow running more in parallel without hurting the CI CPUs.
The default speed is quite fast anyway for a single test run, so in the bottom line this has better performance.